wiz-admission-controller/templates/cronjobmanager.yaml

```diff
@@ -0,0 +1,186 @@
+              {{- end }}
+                - name: WIZ_ENV
+                  value: {{ coalesce .Values.global.wizApiToken.clientEndpoint .Values.wizApiToken.clientEndpoint | quote }}
+              {{- if or .Values.global.httpProxyConfiguration.enabled .Values.httpProxyConfiguration.enabled }}
+                - name: HTTP_PROXY
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.proxySecretName" . | trim }}
+                      key: httpProxy
+                      optional: false
+                - name: HTTPS_PROXY
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.proxySecretName" . | trim }}
+                      key: httpsProxy
+                      optional: false
+                - name: NO_PROXY
+                  valueFrom:
+                    secretKeyRef:
+                      name: {{ include "wiz-admission-controller.proxySecretName" . | trim }}
+                      key: noProxyAddress
+                      optional: false
+              {{- end }}
+              {{- if .Values.logLevel }}
+                - name: LOG_LEVEL
+                  value: {{ .Values.logLevel }}
+              {{- end }}
+              {{- with .Values.podCustomEnvironmentVariables }}
+                {{- toYaml . | nindent 14 }}
+              {{- end }}
+              {{- with .Values.global.podCustomEnvironmentVariables }}
+                {{- toYaml . | nindent 14 }}
+              {{- end }}
+              {{- if .Values.podCustomEnvironmentVariablesFile }}
+                - name: CLI_ENV_FILE
+                  value: {{ .Values.podCustomEnvironmentVariablesFile }}
+                - name: USE_CLI_ENV_FILE
+                  value: "true"
+              {{- end }}
+                - name: WIZ_RUNTIME_METADATA_POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: WIZ_RUNTIME_METADATA_NODE_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: spec.nodeName
+                - name: K8S_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: WIZ_TERMINATION_GRACE_PERIOD
+                  value: "{{ .Values.global.podTerminationGracePeriodSeconds }}s"
+              resources:
+                {{- toYaml .Values.resources | nindent 16 }}
+              volumeMounts:
+                - mountPath: /var/server-certs
+                  name: server-certs
+                  readOnly: true
+              {{- with .Values.customVolumeMounts }}
+                {{- toYaml . | nindent 14 }}
+              {{- end }}
+              {{- with .Values.global.customVolumeMounts }}
+                {{- toYaml . | nindent 14 }}
+              {{- end }}
+          volumes:
```

Comment:
Do we really need these certs?

Reply:
Actually no, we needed it for the health server, but we don't need it eventually so I'm removing it